### PR TITLE
Add analytics

### PR DESCRIPTION
**Pull request by CannonLock**

### CODE REVIEW

**Comment by CannonLock** [comment]
@jasoncpatton Can you please check my changes in commit "Adding Check for Dev or Production build"

File: script/cibuild [hunk 1]
```diff
@@ -1,4 +1,10 @@
 #!/bin/sh
 set -e
 
-bundle exec jekyll build
+if [ $1 = "production" ]; then
```
Comment (jasoncpatton) [inline script/cibuild:4]:
```suggestion
if [ "$1" = "production" ]; then
```

File: script/cibuild [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
+  JEKYLL_ENV=production bundle exec jekyll build
+fi;
+
+if [ $1 = "development" ]; then
```
Comment (jasoncpatton) [inline script/cibuild:8]:
```suggestion
if [ "$1" = "development" ]; then
```

**Review by jasoncpatton — CHANGES_REQUESTED** [review]
`cibuild` will only run `jekyll build` if `development` or `production` are provided as arguments. Should this instead be that the `development` build is done by default unless `production` is passed in?

File: script/cibuild (hunk 1 above)
Comment (CannonLock) [reply script/cibuild:4]: @jasoncpatton Yes, that is a good call. Thank you for the suggestion!

File: script/cibuild [hunk 3]
```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
 set -e
 
-bundle exec jekyll build
+if [ $1 = "production" ]; then
```
Comment (jasoncpatton) [inline script/cibuild]: This disappeared in new commit :)
```suggestion
if [ "$1" = "production" ]; then
```

**Review by jasoncpatton — CHANGES_REQUESTED** [review]

File: script/cibuild (hunk 3 above)
Comment (jasoncpatton) [reply script/cibuild]: Whoops, spoke too soon!

**Review by jasoncpatton — APPROVED** [review]
Not knowing exactly how jekyll works, I assume https://github.com/CHTC/chtc-website-source/blob/783206e76cdfcea3b33ac3ceef0ce15a9de1d16a/.github/workflows/build_deploy.yml#L25 only runs if the first build works. So this LGTM.

**Pull request merged**